src/main/resources/avro/variants.avdl

```diff
@@ -41,7 +41,30 @@ record VariantSetMetadata {
 }
 
 /**
-`Variant` and `CallSet` both belong to a `VariantSet`.
+Method used for variant calling
+*/
+record CallingMethod {
```

Comment:
I am not sure if it matters either way. It just describes meta information. If you insist, I could add `array<CallingMethod> callingMethods` to `VariantSet`.